responses/__init__.py

```diff
@@ -418,6 +418,22 @@ def get_response(self, request):
             preload_content=False,
         )
 
+    def __str__(self):
+        return str(
+            json_module.dumps(
```

Comment:
Could we include a prefix so that it is clearer that the object is not actually a dict/json? Something like `f"Response(url={self.url} status={self.status} content_type={self.content_type} headers={json.dumps(self.headers)})"` could work and might be a way to approach repr too.

Reply:
I like this approach, perhaps we can implement `__repr__` like this and not implement `__str__`, we just leave the same result for both. I think this could work

Reply:
I was wondering if, perhaps, it is a good idea to have this representation for `__str__` and another one similar but including `body` for `__repr__`, like this we would have everything to recreate the same object as is expected for `__repr__`. `body` could be large to add it always for `__str__`